types/ember-data/index.d.ts

```diff
@@ -86,15 +86,15 @@ export namespace DS {
         modelName: K,
         options: RelationshipOptions<ModelRegistry[K]> & Sync
     ): Ember.ComputedProperty<
-        ModelRegistry[K],
-        ModelRegistry[K] | PromiseObject<ModelRegistry[K]>
+        ModelRegistry[K] | null,
```

Comment:
exploring this in the light of the https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52241#issuecomment-816168278

I believe this is wrong, cause In runtime `DS.belongsTo` does not result to `null`? @wagenet